src/r2mm/manager/ModLinker.ts

```diff
@@ -85,28 +104,36 @@ export default class ModLinker {
-                        if (file.toLowerCase() !== 'mods.yml') {
-                            try {
-                                const gameDirFilePath = path.join(installDirectory, file);
-                                const profileDirFilePath = path.join(profile.getPathOfProfile(), file);
-                                if (!(await this.isFileIdentical(profileDirFilePath, gameDirFilePath))) {
-                                    await fs.copyFile(profileDirFilePath, gameDirFilePath);
-                                    const profileDirFileStat = await fs.stat(profileDirFilePath);
-                                    await fs.setModifiedTime(gameDirFilePath, profileDirFileStat.mtime);
-                                }
-                                newLinkedFiles.push(gameDirFilePath);
-                            } catch (e) {
-                                const err: Error = e as Error;
-                                throw new FileWriteError(
-                                    `Couldn't copy file ${file} to ${game.displayName} directory`,
-                                    err.message,
-                                    `Try running ${ManagerInformation.APP_NAME} as an administrator`
-                                )
+                        try {
+                            const targetDir = ModLinker.getRootFilesDestination(game, file, installDirectory);
+                            if (targetDir === null) continue;
+
+                            const gameDirFilePath = path.join(targetDir, file);
+                            const profileDirFilePath = path.join(profile.getPathOfProfile(), file);
+
+                            if (!(await this.isFileIdentical(profileDirFilePath, gameDirFilePath))) {
+                                await fs.copyFile(profileDirFilePath, gameDirFilePath);
+                                const profileDirFileStat = await fs.stat(profileDirFilePath);
+                                await fs.setModifiedTime(gameDirFilePath, profileDirFileStat.mtime);
                             }
+                            newLinkedFiles.push(gameDirFilePath);
+                        } catch (e) {
+                            const err: Error = e as Error;
+                            throw new FileWriteError(
+                                `Couldn't copy file ${file} to ${game.displayName} directory`,
+                                err.message,
+                                `Try running ${ManagerInformation.APP_NAME} as an administrator`
+                            )
                         }
                     } else {
                         if ((await fs.lstat(path.join(profile.getPathOfProfile(), file))).isDirectory()) {
-                            if (!["bepinex", "bepinex_server", "mods", "melonloader", "plugins", "userdata", "_state", "userlibs", "qmods"].includes(file.toLowerCase())) {
+                            const exclusionsList = [
```

Comment:
You could declare this at the top of the file outside of the class if that's easier. It wouldn't need to be recreated every time this is hit.

Reply:
This was a part of #1079 but yeah agreed, didn't end up making it a blocker however.